test/e2e/workflow/deployment_test.go

```diff
@@ -438,6 +438,57 @@ var _ = Describe("NetworkAddonsConfig", func() {
 					}, 5*time.Minute, time.Second).Should(BeNil(), "Timed out waiting for nmstate-operator daemonset")
 				})
 
+				Context("when NAC is updated", func() {
```

Comment:
Although `NAC` is a cool name for NetworkAddonsConfig, we don't really use it so it might create future confusion. I would suggest using `NetworkAddonsConfig`/`CNAO CR` instead

Reply:
[Done](https://github.com/kubevirt/cluster-network-addons-operator/compare/08d160a48ed4b089cc654761fd1112b3bb3a3e68..35f776d9be08fce4039c5feeb2a2cfa0f8faec9e)